### PR TITLE
Feature/change titles

### PR DESCRIPTION
**Pull request by JulieAtInnoactive**
### Description
changed labels

Links to https://github.com/Innoactive/Creator/pull/256

image

**Fixes**: # (issue)

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

**Test Configuration**:

### Checklist

- [ ] My code follows the [Coding Conventions](#coding-conventions)
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by tomwim — APPROVED** [review]

**Review by SimonTheSourcerer — APPROVED** [review]
If you dont squash this, I will personally exclude you from the next round of cake!

**Review by SimonTheSourcerer — APPROVED** [review]

**Review by tomwim — APPROVED** [review]

File: Runtime/Conditions/TimeoutCondition.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public class EntityData : IConditionData
             /// The delay before the condition completes.
             /// </summary>
             [DataMember]
-            [DisplayName("Wait for seconds")]
+            [DisplayName("Wait (in seconds)")]
```
Comment (Gusinuhe) [inline Runtime/Conditions/TimeoutCondition.cs:24]: Perhaps something clearer like "Wait (time in seconds)" 🤔

**Review by vued — COMMENTED** [review]
(in seconds) vs (in sec.) 
I assume this is because of the length of the description. 
We don't have a fixed length since the valid length depends on where the window is put and what screen is used. We can consider a guideline for length of labels for conditions/behaviors.
I prefer consistency. How about shortening this in all behaviors/conditions which need a metric ?
Maybe even shorten to "(in s)" - don't know if this (internatianally) clear enough, though...

File: Runtime/Conditions/TimeoutCondition.cs (hunk 1 above)
Comment (JulieAtInnoactive) [reply Runtime/Conditions/TimeoutCondition.cs:24]: I would like to keep it consistent <shortest name possible> (in <unit>)

**Review by JulieAtInnoactive — COMMENTED** [review]
I made the display of units consistent acrross all behaviors (see move behavior)

**Review by Gusinuhe — APPROVED** [review]

File: Runtime/Conditions/TimeoutCondition.cs (hunk 1 above)
Comment (Gusinuhe) [reply Runtime/Conditions/TimeoutCondition.cs:24]: Alright, but this does not make much sense 😕

**Comment by tomwim** [comment]
Needs rebase and renaming of commit `consistent unit`

**Review by SimonTheSourcerer — APPROVED** [review]

**Review by tomwim — APPROVED** [review]

**Review by SimonTheSourcerer — APPROVED** [review]

**Pull request merged**